### PR TITLE
Add yarn in add project page

### PR DESCRIPTION
**Pull request by MichanNan**
please check

### CODE REVIEW

File: components/ProjectForm/index.js [hunk 1]
```diff
@@ -179,6 +203,7 @@ export default function ProjectForm({
             {buttonContentRight}
           </StyledButton>
         </RowSection>
+        <label htmlFor="brand"></label>
```
Comment (rueveydaakuel) [inline components/ProjectForm/index.js:428]: Is this label used? It seems to be empty

File: components/YarnItem/index.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+  function handleInputChange(event) {
+    const { name, value } = event.target;
+    setYarnData((prevData) => {
+      const updatedData = [...prevData];
+      updatedData[yarnCount] = {
+        ...updatedData[yarnCount],
+        [name]: value,
+      };
+      console.log(yarnData);
```
Comment (rueveydaakuel) [inline components/YarnItem/index.js]: I am not sure if this console.log is allowed to stay

**Review by rueveydaakuel — COMMENTED** [review]
Your code looks very good in my opinion 🙂 I wasn't sure about some places, so I left two comments there.

File: components/YarnItem/index.js (hunk 2 above)
Comment (MichanNan) [reply components/YarnItem/index.js]: Thanks Rüveyda, i delete these code. They are unnecessary 😄

**Comment by sebastianwel** [comment]
Hey Nan,
your code looks good and the update is fine in my opinion, so I will approve your code 😄
One thing however I saw, when trying to view your vercel-project is that the app is not loading there:
image

Not sure if it's my browser, so please check before you post it into QA! ✅

**Review by sebastianwel — APPROVED** [review]

**Comment by MichanNan** [comment]
> Hey Nan, your code looks good and the update is fine in my opinion, so I will approve your code 😄 One thing however I saw, when trying to view your vercel-project is that the app is not loading there: image
> 
> Not sure if it's my browser, so please check before you post it into QA! ✅

Thanks Sebastian, i will check it!

**Pull request merged**